components/dropdown/README.md

```diff
@@ -1,10 +1,12 @@
+
 # Dropdowns
-## Dropdown Openers
+A Dropdown is a button that opens a floating container to offer menu items or other content.
 
-### d2l-dropdown
+## Opener: Dropdown [d2l-dropdown]
```

Comment:
Not sure how I feel about these titles, I thought some method of grouping openers/content would be ideal. Maybe these get split into a separate page?

Reply:
Yeah for sure - I like that you added something to distinguish the type. Can probably leave as is for now and see what Jeff thinks.